cli/flox/doc/flox-containerize.md

```diff
@@ -46,10 +50,6 @@ Running the container non-interactively with `docker run <container id>`
 allows you to run a command within the container without launching a subshell,
 similar to `flox activate --`.
 
-**Note**:
-The `containerize` command is currently **only available on Linux**.
-The produced container however can also run on macOS.
```

Comment:
I also need to remove the warning from https://flox.dev/docs/tutorials/sharing-environments/?h=container#sharing-with-containers and might as well do https://github.com/flox/product/issues/767 at the same time.